packages/cli/src/lib/webdriver.test.ts

```diff
@@ -101,4 +104,15 @@ describe('startDriver', () => {
     assert.isObject(timeoutValue);
     assert.deepEqual(timeoutValue.script, 10000000);
   });
+
+  it('test headless option in selenium-webdriver < 4.17.0', async () => {
```

Comment:
nit - we aren't testing if the `options.headless()` call passes the right flag to the Driver, we are just asserting that the function was called when using < 4.17.0 - 
```suggestion
  it('invokes `options.headless()` on versions of selenium-webdriver < 4.17.0', async () => {
```